### PR TITLE
Refresh boundaries

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/orgs/tasks.py [hunk 1]
```diff
@@ -29,3 +29,14 @@ def build_boundaries():
                 logger.debug("=" * 40)
                 org.build_boundaries()
     logger.debug("Task: build_boundaries took %ss" % (time.time() - start))
+
+
+@shared_task(track_started=True, name='fetch_poll')
+def rebuild_org_boundaries(org_id):
+    try:
+        # get our poll
```
Comment (nicpottier) [inline dash/orgs/tasks.py]: typo

File: dash_test_runner/settings.py [hunk 2]
```diff
@@ -279,7 +279,7 @@
 # Async tasks with django-celery
 # ----------------------------------------------------------------------------
 
-CELERY_RESULT_BACKEND = 'database'
+CELERY_RESULT_BACKEND = 'redis://'
```
Comment (nicpottier) [inline dash_test_runner/settings.py]: What's the reasoning here?

Comment (norkans7) [inline dash_test_runner/settings.py]: There were an error of missing dependency sqlalchemy so I thought it is not worth a new dependency just for testing I switched this setting to redis which is already used and installed

Comment (nicpottier) [inline dash_test_runner/settings.py]: I don't think that's the DB we want to use though: (this defaults to localhost/0)
  http://celery.readthedocs.org/en/latest/configuration.html#conf-redis-result-backend

Comment (norkans7) [inline dash_test_runner/settings.py]: these settings are used just for testing I guess that is fine

**Comment by nicpottier** [comment]
Looks good.

**Pull request merged**